index.html

```diff
@@ -3,10 +3,116 @@
+      </li>
+      <li class="advantages-item">
+        <img class="image" src="photo-2.jpg" alt="Фото настоящего каньона">
+        <h3 class="advantages-title">Там есть мост дьявола</h3>
+        <span class="advantages-description">— №2 —</span>
+        <p class="advantages-description">Да, по нему можно пройти! Если вы осмелитесь, разумеется</p>
+      </li>
+      <li class="advantages-item">
+        <h3 class="advantages-title">Небольшая площадь</h3>
+        <span class="advantages-description">— №3 —</span>
+        <p class="advantages-description">Все достопримечательности находятся очень близко</p>
+      </li>
+      <li class="advantages-item">
+       <h3 class="advantages-title">Красивая дорога</h3>
+        <span class="advantages-description">— №4 —</span>
+        <p class="advantages-description">Ехать в Седону из Лас-Вегаса совсем не скучно!</p>
+      </li>
+      <li class="advantages-item">
+        <h3 class="advantages-title">Мало туристов</h3>
+        <span class="advantages-description">— №5 —</span>
+        <p class="advantages-description">Большинство едет в Гранд Каньон и толпится там</p>
+      </li>
+    </ul>
+  </section class="search">
+    <h3 class="search-title">Заинтересовались?</h3>
+    <p class="search-description">Укажите предполагаемые даты поездки,и мы покажем вам лучшие предложения гостиниц в Седоне</p>
+    <button class="search-button popup" type="button"><span class="search-">Поиск гостиницы В седоне</span></button> 
+  </section>
+  <section class="subscribe">
+    <h3 class="subscribe-title">Подпишитесь на рассылку</h3>
+    <p class="subscribe-description">Только полезная информация и никакого спама, честное бойскаутское!</p>
+    <form class="subscribe-form" action="https://echo.htmlacademy.ru/" method="post">
+    <label class="label-form" for="email"></label>
+    <input class="subscribe-input" name="email" id="email" type="email" placeholder="Ваш e-mail" required>
+    <button class="subscribe-button" type="submit">Подписаться</button>
+    </form>
+  </section>
+  </main>
+  <footer class="page-footer">
+    <section class="contacts">
+      <ul class="contacts-list">
+        <li class="VK">
+          <a class="contacts-link-VK" href="#"><span class="visually-hidden">Вконтакте</span><img class="logo-VK" src="" alt="Лого ВК"></a>
+        </li>
+       <li class="Telegram">
+          <a class="contacts-link-Telegram" href="#"><span class="visually-hidden">Телеграмм</span><img class="logo-Telegram" src="" alt="Лого Телеграмм"></a>
+        </li>
+        <li class="Youtube">
+          <a class="contacts-link-Youtube" href="#"><span class="visually-hidden">Ютуб</span><img class="logo-Youtube" src="" alt="Лого Ютуб"></a>
+       </li>
+     </ul>
+    </section>
+    <address class="contacts-number">
+      <a class="contacts-number-link" href="tel:+7 (812) 812-12-12">+7 (812) 812-12-12</a>
```

Comment:
Чтобы не было ошибки валидатора
```suggestion
      <a class="contacts-number-link" href="tel:+78128121212">+7 (812) 812-12-12</a>
```